src/h264/h264.h

```diff
@@ -180,8 +226,328 @@ class H264SeqParameterSet
 	DWORD			frame_crop_right_offset = 0;
 	DWORD			frame_crop_top_offset = 0;
 	DWORD			frame_crop_bottom_offset = 0;
-	//bool			vui_parameters_present_flag = false;
+	bool			vui_parameters_present_flag = false;
 	bool			separate_colour_plane_flag = 0;
+
+	struct HrdParameters
+	{
+		uint32_t              cpb_cnt_minus1 = 0;
+		uint32_t              bit_rate_scale = 0;
+		uint32_t              cpb_size_scale = 0;
+		std::vector<uint32_t> bit_rate_value_minus1;
+		std::vector<uint32_t> cpb_size_value_minus1;
+		std::vector<uint32_t> cbr_flag;
+		uint32_t              initial_cpb_removal_delay_length_minus1 = 0;
+		uint32_t              cpb_removal_delay_length_minus1 = 0;
+		uint32_t              dpb_output_delay_length_minus1 = 0;
+		uint32_t              time_offset_length = 0;
+		bool ParseHRDParams(RbspBitReader &r)
```

Comment:
keep naming consistency
```suggestion
		bool Decode(RbspBitReader &r)
```